vimtutor.bat

```diff
@@ -10,7 +10,20 @@
 :: When that also fails, it uses the English version.
 
 :: Use Vim to copy the tutor, it knows the value of $VIMRUNTIME
-FOR %%d in (. "%TMP%" "%TEMP%") DO IF EXIST %%d\nul SET TUTORCOPY=%%d\$tutor$
+FOR %%d in (. %TMP% %TEMP%) DO (
+call :test_dir %0 %%d
+IF NOT ERRORLEVEL 1 GOTO dir_ok
+)
+
+echo No working directory is found
+GOTO end
+
+:test_dir
```

Comment:
How about making the label more explanatory? (e.g. `:test_dir_writable`)
Or how about adding a comment?

Reply:
I think changing the label would be clearer. I prefer better names to comments.